build/package/windows/kbc.wxs

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<?ifndef ProductVersion?>
+<?error ProductVersion property not defined?>
+<?endif?>
+
+<!-- Define a unique UpgradeCode per platform -->
+<?if $(var.Platform) = "x64"?>
+<?define InstallerVersion = "200"?>
+<?define UpgradeCode = "25389d03-f8c7-415e-9755-ce0666951c1f"?>
+<?define ProgramFilesFolder = "ProgramFiles64Folder"?>
+<?elseif $(var.Platform) = "x86"?>
+<?define InstallerVersion = "200"?>
+<?define UpgradeCode = "562DED1E-651A-4703-A979-9EBDC4CB2530"?>
+<?define ProgramFilesFolder = "ProgramFilesFolder"?>
+<?elseif $(var.Platform) = "arm64"?>
+<?define InstallerVersion = "500"?>
+<?define UpgradeCode = "740FD01E-BC4B-4C79-B870-27208CF44379"?>
+<?define ProgramFilesFolder = "ProgramFiles64Folder"?>
+<?elseif $(var.Platform) = "arm"?>
+<?define InstallerVersion = "500"?>
+<?define UpgradeCode = "FFBCF74F-970D-478B-888C-1B352F546DF3"?>
+<?define ProgramFilesFolder = "ProgramFilesFolder"?>
```

Comment:
Asi nebude uplne jednoduche spravit `msi`, aj pre ostatne platformy, ktore podporujeme (arm) ? (predpokladam, ze nie)

Reply:
No jo vlastně 🤔 tady se předpokládá vytváření msi pro ty ostatní platformy ale stahuje se jen amd_64 exe. A dívám se, že Github má taky deg'finovaný tady všechny tři, ale msi generuje jen jedno. Tak imho tady by se dala jen změnit platforma: https://github.com/keboola/keboola-as-code/blob/a88cf33d459569b3945d0037a76b1eca2acde564/build/package/windows/msi.wixproj#L5-L5 a při troše štěstí to projde. V horším případě by se to muselo buildnout na odpovídajícím vmku. Google k tomu nic moc lepšího neříká.

Reply:
Asi sprav issue, tipujem ze sa to bude muselo buildnut na danej platforme (v tom pripade by sme to neriesili).